README.md

```diff
@@ -84,6 +88,16 @@ create_list(3, :comment, attrs)
 fields_for(:comment, attrs)
 ```
 
+## Where to put your factories
+
+We recommend starting by creating one factory module (such as MyApp.Factory) in
+`lib/my_app/factory.ex` and putting all factory definitions in that module.
+
+Later on you can easily create different factories if you need to hook them up
+to different repos, if your factory module is getting to big, or if you have
```

Comment:
Thoughts on ending the first sentence at `to different repos.` Then maybe something along the lines of: `If you prefer multiple factories for any reason you can create another factory module in the same directory`.

I think it would help if `create another factory module` was more clearly defined or an example was given to avoid confusion.